TASVideos/Pages/Profile/Settings.cshtml

```diff
@@ -92,37 +92,33 @@
 		</column>
 	</row>
 	<hr />
-	<row>
-		<column md="6">
-			<label asp-for="Settings.Avatar"></label>
-			<input type="text" asp-for="Settings.Avatar" class="form-control" />
-		</column>
-		<column id="avatar-preview" md="6" class="mt-md-0 mt-3" show="!string.IsNullOrWhiteSpace(Model.Settings.Avatar)">
-			<label>Preview</label>
-			<button id="preview-btn" type="button" class="btn btn-secondary btn-sm">
-				<span class="fa fa-refresh"></span>
-			</button><br />
-			<img id="avatar-img" src="@Model.Settings.Avatar" class="float-start m-2" />
-			<small>
-				@await Component.RenderWiki("System/AvatarRequirements")
-			</small>
-		</column>
-	</row>
-	<row permission="UseMoodAvatars">
-		<column md="6">
-			<label asp-for="Settings.MoodAvatar"></label>
-			<input type="text" asp-for="Settings.MoodAvatar" class="form-control" />
-			<a asp-page="/Forum/MoodReport" asp-route-username="@Model.Settings.Username">Check Mood Avatars</a>
-		</column>
-		<column md="6">
-			<small>
-				@await Component.RenderWiki("System/MoodAvatarRequirements")
-			</small>
-		</column>
-	</row>
 	<fullrow class="mt-3">
+		<label asp-for="Settings.Avatar"></label>
+		<input type="text" asp-for="Settings.Avatar" class="form-control" />
+	</fullrow>
+	<fullrow class="mt-2" id="avatar-preview" show="!string.IsNullOrWhiteSpace(Model.Settings.Avatar)">
```

Comment:
`show` doesn't work because tag helpers don't seem to print it out

![3](https://user-images.githubusercontent.com/7092625/235429431-fb5e104b-ee1a-44fa-aeb0-66468e3db9c9.png)